README.md

```diff
@@ -1,19 +1,27 @@
 # Innkeeper
 
-A route management API for [Skipper](https://github.com/zalando/skipper)
+Innkeeper is a simple route management API for [Skipper](https://github.com/zalando/skipper)
 
 When a new instance of Skipper is started, it will connect to innkeeper, ask for all the routes and initialize it's own data structures.
```

Comment:
"...of Skipper, configured to fetch routes from Inkeeper, it will request all the routes..."